blog-post.html

```diff
@@ -1,12 +1,162 @@
-<!DOCTYPE html>
+<!DOCTYPE HTML>
 <html>
-  <head>
-    <title>Blog Post</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="main"></section>
-  </body>
-</html>
+
+<head>
+  <title>Motherflippin Homepage</title>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+  <link href="style.css" rel="stylesheet" />
+</head>
+
+<body id="post">
+
+  <!-- header section -->
+  <div class="item header">
+    <a href="index.html">
+      <h1>
+        B L O G &nbsp P O S T
+      </h1>
+    </a>
+    <div class="subhead">
+      THE PAGE FOR THE POST.
+
+      <div class=headrln>
+        <br> <br />

```

Comment:
Rather then using `<br />` tags here, try using the css property of `margin: (x)`. Margins allow you to be more specific and the `<br />` tags are better used within text content like a `<p>` tag